### PR TITLE
✅ Add unit tests for amp-facebook-page

### PR DESCRIPTION
**Pull request by caroqliu**

### CODE REVIEW

File: extensions/amp-facebook-page/0.1/test/test-amp-facebook-page.js [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+      facebook(win, {
+        href,
+        width: 100,
+        height: 100,
+      });
+      const container = doc.body.getElementsByClassName('fb-page')[0];
+      expect(container).not.to.be.undefined;
+      expect(container.getAttribute('data-href')).to.equal(href);
+      expect(container.getAttribute('data-tabs')).to.equal('undefined');
```
Comment (caroqliu) [inline extensions/amp-facebook-page/0.1/test/test-amp-facebook-page.js:253]: It's not great that many of these are `undefined` --- we should probably fix these to utilize the [documented defaults](https://amp.dev/documentation/components/amp-facebook-page/#attributes_1), but was thinking this might be better in a separate PR so we don't accidentally break something while we introduce tests...

**Review by krdwan — APPROVED** [review]
